extensions/visualizations/visualizations.html

```diff
@@ -6,14 +6,14 @@
   <strong><[options.title]></strong>
   <table class="table">
     <tr>
-      <th ng-repeat="column_header in options.column_headers">
-        <[column_header]>
-      </th>
+      <th><[options.column_headers[0]]></th>
```

Comment:
Perhaps this (and the one below) should still be ng-repeats?

Reply:
I changed it because I think it helps readability to see that there are exactly two columns that are rendered, and that they correspond to `<[data.answer]>` and `<[data.frequency]>`. I'd probably use ng-repeats if there were a nice programmatic way to access the data elements as well.

Reply:
Hm. If we're going to always assume two columns, should we perhaps change the schema so that only the name of the first column is specified? (The second one always seems to be "Count", so that probably doesn't need to be parameterized.) Otherwise column_headers implies it can be a list of arbitrary length (hence the ng-repeat suggestion for the case where there are != 2 columns, but I take your point that this probably isn't going to occur).

OK for this to be in a separate PR, though.

Reply:
Yeah, I'll just do that in a separate PR.